src/defaultsettings.cpp

```diff
@@ -199,63 +203,59 @@ void set_default_settings(Settings *settings)
 	settings->setDefault("fallback_font_shadow", "1");
 	settings->setDefault("fallback_font_shadow_alpha", "128");
 
-	std::stringstream fontsize;
-	fontsize << TTF_DEFAULT_FONT_SIZE;
+	std::string font_size_str = to_string(TTF_DEFAULT_FONT_SIZE);
```

Comment:
1. `std::to_string` is C++11, `::to_string` is defined in `util/string.h`.
  2. It's `TOSTRING()`, but yes, I probably should.